apps/consulting/posts/improvements-to-bsr-matrix-multiplication-in-pytorch.md

```diff
@@ -0,0 +1,292 @@
+---
+title: "Improvements to BSR Matrix Multiplication in PyTorch"
+published: October 1, 2023
+author: andrew-james
+description: 'Quansight engineers have implemented several new kernels for matrix multiplication
+involving sparse and dense arguments. These implementations, available in PyTorch 2.1,  offer a
+significant performance improvement over those available in previous versions of PyTorch. In
+addition, they have the unique property that they are implemented using `triton`, however they are
+not the product of `torch.compile`.'
+category: [Open Source Software, PyTorch, Triton, GPU]
+---
+<base target="_blank" />
+
+Quansight engineers have implemented several new kernels for matrix multiplication involving sparse
+and dense arguments. These implementations, available in PyTorch 2.1,  offer a significant
+performance improvement over those available in previous versions of PyTorch. In addition, they have
+the unique property that they are implemented using `triton`, however they are not the product of
```

Comment:
What is triton? Why is it surprising that torch.compile is not involved?

Reply:
This part will come out of the intro, see below.